db/migrate/20150405012126_remove_approval_status_from_reservation.rb

```diff
@@ -0,0 +1,35 @@
+class RemoveApprovalStatusFromReservation < ActiveRecord::Migration
+  def change
+    Reservation.all.each do |res|
+      if res.approval_status == 'approved'
+        res.flag(:request)
+      elsif res.approval_status == 'missed_and_emailed'
+        res.flag(:missed_email_sent)
+      end
+
+      if res.checked_in 
+        res.status = 'returned'
+        if res.checked_in > res.due_date
```

Comment:
this should be (I think) `res.checked_in > res.due_date + 1.day` since patrons are allowed to return equipment *on* the due date itself (so if `due_date = 2015-01-01` and `checked_in = 2015-01-01 12:00`, we want it to still not be `overdue`).